docs/guide/conditional-fields.md

```diff
@@ -30,20 +33,231 @@ add: {
+    $lte: 65
+  },
+  email: {
+    $ne: ''  // Not empty string (implies existence for practical purposes)
+  },
+  status: {
+    $in: ['active', 'verified'],
+    $ne: 'suspended'
+  }
+}
+```
+
+::: info Negation without `$not`
+While there's no `$not` operator, you can achieve negation using other operators:
+- Use `$ne` instead of `$not: 'value'`
+- Use `$nin` instead of `$not: { $in: ['a', 'b'] }`
+- Use `$exists: false` instead of `$not: { $exists: true }`
+:::
+
+::: info
+When using `$exists`, a value of `true` checks that the field has a value (not null or undefined), while `false` checks that the field is null or undefined.
+:::
+
 ## Nested Conditional Fields
 
 You can also create a hierarchy of conditional fields where child field visibility depends on a parent field. While the display of this parent field, in turn, is influenced by the value of another field in the schema. The child fields will remain hidden until both the parent field conditional display criteria is satisfied and the value of the parent field fulfills the child field conditional.
 
+## Accessing parent schema fields
+
+When working with nested schemas (like `array` or `object` fields), conditional fields can access values from parent schema fields using the `following` property with the same `<` prefix syntax used for dynamic choices.
+
+```javascript
+// Example with nested object field
+add: {
+  projectType: {
+    label: 'Project Type',
+    type: 'select',
+    choices: [
+      { label: 'Internal', value: 'internal' },
+      { label: 'Client', value: 'client' }
+    ]
+  },
+  projectDetails: {
+    label: 'Project Details',
+    type: 'object',
+    fields: {
+      add: {
+        clientBudget: {
+          label: 'Client Budget',
+          type: 'integer',
+          // 👇 Access parent field value
+          if: {
+            '<projectType': 'client'
+          }
+        },
+        internalCostCenter: {
+          label: 'Cost Center',
+          type: 'string',
+          if: {
+            '<projectType': 'internal'
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Advanced parent field access with dot notation
+
+You can use dot notation to access nested properties, array elements, and special properties like `length`:
+
+```javascript
+add: {
+  tasks: {
+    label: 'Tasks',
+    type: 'array',
+    fields: {
+      add: {
+        title: {
+          label: 'Task Title',
+          type: 'string'
+        },
+        priority: {
+          label: 'Priority',
+          type: 'select',
+          choices: [
+            { label: 'Low', value: 'low' },
+            { label: 'High', value: 'high' }
+          ]
+        }
+      }
+    }
+  },
+  projectSettings: {
+    label: 'Project Settings',
+    type: 'object',
+    fields: {
+      add: {
+        // Show only when there are tasks
+        taskSummary: {
+          label: 'Task Summary',
+          type: 'string',
+          if: {
+            '<tasks.length': { $gt: 0 }
```

Comment:
Add `following` for all examples here. For fields with type `string` or `slug`, add `followingExclude: ['<tasks']` as well.